src/test/java/fr/techad/edc/popover/swing/example/Main.java

```diff
@@ -1,6 +1,7 @@
 package fr.techad.edc.popover.swing.example;
 
 import fr.techad.edc.client.model.InvalidUrlException;
+import fr.techad.edc.popover.model.PopoverPlacement;
```

Comment:
Not use, either to remove this import or you add the popover placement in the configuration (with default value)